tests/wallet/test_wallet.py

```diff
@@ -562,6 +562,8 @@ async def test_wallet_tx_reorg(self, two_wallet_nodes):
         funds = sum(
             [calculate_pool_reward(uint32(i)) + calculate_base_farmer_reward(uint32(i)) for i in range(1, num_blocks)]
         )
+        # Waits a few seconds to receive rewards
+        await asyncio.sleep(5)
```

Comment:
is this wall-clock time or is the clock simulated in the tests?

Reply:
wall clock time, but it doesn't seem to help. Still flaky test